HOW_TO_USE.md

```diff
@@ -37,15 +37,15 @@ Dont't know what these stand for? No problem, following are the steps. 
     - To only fetch the issues and store without tweeting
 
       ```
-      python first_timers/run.py --creds-path credentials.json --db-path data/db.json --only-save
+      python first_timers/run.py --creds-path credentials.json --db-path data/db.json --only-save --create
```

Comment:
We do not want to update every command with create flag.
Remove `--create` from this